sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1504 @@
+    "|**country_or_region_for_holidays**|The country/region used to generate holiday features. These should be ISO 3166 two-letter country/region codes (i.e. 'US', 'GB').|\n",
+    "\n",
+    "#### Using lags\n",
+    "This training is also using the **target lags** which are lagged values of the target variable. Since we generate lags with respect to horizon, we therefore must still specify the `forecast_horizon` that the model will learn to forecast. The `target_lags` keyword specifies how far back we will construct the lags of the target variable. The `target_lags` is set to auto in this notebook. Therefore, this value will be computed automatically.\n",
+    "\n",
+    "This notebook uses the `.set_training(blocked_training_algorithms=...)` parameter to exclude some models that take a longer time to train on this dataset.  You can choose to remove models from the blocked_training_algorithms list but you may need to increase the trial_timeout_minutes parameter value to get results.\n",
+    "\n",
+    "To run AutoML, you need to create an `Experiment`. An `Experiment` corresponds to a prediction problem you are trying to solve, while a Run corresponds to a specific approach to the problem."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# choose a name for the run history container in the workspace\n",
+    "if isinstance(TARGET_LAGS, list):\n",
+    "    TARGET_LAGS_STR = (\n",
+    "        \"-\".join(map(str, TARGET_LAGS)) if (len(TARGET_LAGS) > 0) else None\n",
+    "    )\n",
+    "else:\n",
+    "    TARGET_LAGS_STR = TARGET_LAGS\n",
+    "\n",
+    "experiment_desc = \"diff-{}_lags-{}_STL-{}\".format(\n",
+    "    DIFFERENCE_SERIES, TARGET_LAGS_STR, STL_TYPE\n",
+    ")\n",
+    "\n",
+    "# General job parameters.\n",
+    "max_trials = 5\n",
+    "exp_name = \"univariate_recipe_{}\".format(experiment_desc)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670411765327
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### 4.2 Create the AutoML forecasting job"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Create the AutoML forecasting job with the related factory-function.\n",
+    "\n",
+    "forecasting_job = automl.forecasting(\n",
+    "    compute=cluster_name,\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    target_column_name=TARGET_COLNAME,\n",
+    "    primary_metric=\"NormalizedRootMeanSquaredError\",\n",
+    "    n_cross_validations=3,\n",
+    "    enable_model_explainability=True,\n",
+    "    tags={\"my_custom_tag\": \"My custom value\"},\n",
+    ")\n",
+    "\n",
+    "# Limits are all optional\n",
+    "forecasting_job.set_limits(\n",
+    "    timeout_minutes=600,\n",
+    "    trial_timeout_minutes=20,\n",
+    "    max_trials=max_trials,\n",
+    "    enable_early_termination=True,\n",
+    ")"
```

Comment:
Let us keep parity with V1 notebook:
timeout_minutes=20,
trial_timeout_minutes=5,
and remove the max_trials